bandwidth/account/client_module.py

```diff
@@ -68,7 +68,7 @@ def _request(self, method, url, *args, **kwargs):
 
     def _check_response(self, response):
         if response.status_code >= 400:
-            if response.headers.get('content-type') == 'application/json':
+            if "application/json" in response.headers.get('content-type'):
```

Comment:
I would prefer to see a startsWith()

Reply:
Though what you have will likely work correctly in any conceivable case

Reply:
How about a try/catch without checking the content type. We’re either returning json or not, if the json parser works then we’re good. Catch any error and return some more context